.github/workflows/deploy-ec2.yaml

```
@@ -0,0 +1,59 @@
+---
+name: Deploy Collector on EC2
+
+'on':
+  schedule:
+    - cron: "0 */1 * * *"
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  check-collector-endpoint:
+    name: 'Check Collector endpoint'
+    runs-on: ubuntu-latest
+
+    outputs:
+      endpoint_status: ${{ steps.collector_sanity_check.outputs.success }}
+
+    steps:
+      - name: 'Check Collector endpoint'
+        id: collector_sanity_check
+
+        run: |
+          set -e
+          response=$(curl -s -o /dev/null -w "%{http_code}" http://claims-collector.cnf-certifications.sysdeseng.com || echo "000")
```

It enables exit on error, but echo masks an error.
```suggestion
          response=$(curl -s -o /dev/null -w "%{http_code}" http://claims-collector.cnf-certifications.sysdeseng.com || echo $?)
```

---

Oh, probably GitHub CI requires '-e' in this case. Just make sure to print the actual error code '$?'.